nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -513,6 +513,12 @@ class OVResultMetatype(OVOpMetatype):
                                   OVConvolutionBackpropDataMetatype,
                                   OVMatMulMetatype]
 
+METATYPES_WITH_WEIGHT_PORT_ID = GENERAL_WEIGHT_LAYER_METATYPES + [OVAddMetatype]
+
+LAYERS_WITH_BIAS_METATYPES = [OVConvolutionMetatype,
```

Comment:
I'm not sure but maybe it is better to use `OPERATIONS_WITH_BIAS_METATYPES`?
By the way, it would be great to add a comment above. Something like this:
```
# Contains the operation metatypes for which bias can be applied.
```